Marquee/Cells/Table Cell/TableViewCell.swift

```diff
@@ -8,18 +8,15 @@
 import UIKit
 import Kingfisher
 
-protocol collectionViewDelegate: AnyObject {
+protocol CollectionViewDelegate: AnyObject {
     func didSelectItem(_ selectedContent: Content)
 }
 
 class TableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
     
     @IBOutlet weak var collectionView: UICollectionView!
-    
     static let identifier = "TableViewCell"
-    
-    var delegate: collectionViewDelegate?
-    
+    var delegate: CollectionViewDelegate?
```

Comment:
It looks like the `CollectionViewDelegate` protocol is conforming to `AnyObject`, is the intention for that to make your `var delegate` weak?

Reply:
I believe my thinking there was that I didn't want to be limited to a UIViewController if in the future I decided to use a UICollectionViewController or UITableViewController. 

Would this be considered bad practice?